spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1214,6 +1214,15 @@ public void testInvalidOptions() {
         .hasMessageContaining("requires enabling Iceberg Spark session extensions");
   }
 
+  @TestTemplate
+  public void testIgnoreInvalidOptions() {
+    Table table = createTable(20);
+    basicRewrite(table)
+        .option("ignore-invalid-options", "true")
+        .option("foobarity", "-5")
+        .execute();
```

Comment:
Should we also test the default behavior of `ignore-invalid-options`, similar to what you did in `TestRewritePositionDeleteFilesAction.testIgnoreInvalidOptions`?